src/font.rs

```diff
@@ -353,6 +353,70 @@ impl Font {
             .guidelines
             .get_or_insert_with(Default::default)
     }
+
+    /// Returns a vector of immutable lib key string references
+    pub fn lib_keys(&self) -> Vec<&String> {
```

Comment:
I don't think there's any real value in returning a `Vec` here. I would have a single function that returns an iterator, and then the user is free to make a vec if they want.